src/dmtcp_coordinator.cpp

```diff
@@ -94,15 +94,16 @@ using namespace dmtcp;
 
```

Comment:
I think this commit ("Added prompt for dmtcp_coordinator.") seems to be about using a different programming style (e.g., lambda function instead of iteration;  'if...else if' instead of 'switch...case').

If this is the case, let's break this out into an independent PR, so it doesn't delay the many other important commits.

If there is something here that changes the semantics, can you isolate that into a separate commit for the current PR, and push the remaining changes into a separate PR?

Thanks.